package.json

```diff
@@ -2,9 +2,6 @@
   "name": "nextjs-project",
   "version": "0.0.0",
   "private": true,
-  "workspaces": [
-    "packages/*"
-  ],
```

Comment:
This no longer matches the pnpm-workspaces file. Preferring the pnpm specific file.